internal/deepprint/deepprint.go

```diff
@@ -45,6 +49,69 @@ func print(w *bufio.Writer, v reflect.Value, visited map[uintptr]bool) {
 	if !v.IsValid() {
 		return
 	}
+
+	// Special case some common types.
+	if v.CanInterface() {
+		switch v.Type() {
+		case reflect.TypeOf(netaddr.IP{}):
```

Comment:
Is reflect.TypeOf(X) done at compile time these days? I thought that bug was still open: https://github.com/golang/go/issues/16869

Pull these out to package variables instead?